bokeh/charts/_charts.py

```diff
@@ -47,7 +47,1621 @@
+        return glyph
+
+    def make_annular(self, source, **kws):
+        """Create a annular wedge glyph and append it to the renderers list.
+
+        Args:
+            source (obj): datasource object containing rect refereces.
+            **kws (refer to glyphs.AnnularWedge for arguments specification details)
+
+        Return:
+            rect: AnnularWedge instance
+        """
+        glyph = AnnularWedge(**kws)
+        self._append_glyph(source, glyph)
+        return glyph
+
+    def make_text(self, source, **kws):
+        """Create a text glyph and append it to the renderers list.
+
+        Args:
+            source (obj): datasource object containing rect references.
+            **kws (refer to glyphs.Text for arguments specification details)
+
+        Return:
+            glyph: Text instance
+        """
+        glyph = Text(**kws)
+        self._append_glyph(source, glyph)
+        return glyph
+
+    def make_scatter(self, source, x, y, markertype, color, line_color=None,
+                     size=10, fill_alpha=0.2, line_alpha=1.0):
+        """Create a marker glyph and appends it to the renderers list.
+
+        Args:
+            source (obj): datasource object containing markers references.
+            x (str or list[float]) : values or field names of line ``x`` coordinates
+            y (str or list[float]) : values or field names of line ``y`` coordinates
+            markertype (int or str): Marker type to use (e.g., 2, 'circle', etc.)
+            color (str): color of the points
+            size (int) : size of the scatter marker
+            fill_alpha(float) : alpha value of the fill color
+            line_alpha(float) : alpha value of the line color
+
+        Return:
+            scatter: Marker Glyph instance
+        """
+        if line_color is None:
+            line_color = color
+
+        _marker_types = OrderedDict(
+            [
+                ("circle", Circle),
+                ("square", Square),
+                ("triangle", Triangle),
+                ("diamond", Diamond),
+                ("inverted_triangle", InvertedTriangle),
+                ("asterisk", Asterisk),
+                ("cross", Cross),
+                ("x", X),
+                ("circle_cross", CircleCross),
+                ("circle_x", CircleX),
+                ("square_x", SquareX),
+                ("square_cross", SquareCross),
+                ("diamond_cross", DiamondCross),
+            ]
+        )
+
+        g = itertools.cycle(_marker_types.keys())
+        if isinstance(markertype, int):
+            for i in range(markertype):
+                shape = next(g)
+        else:
+            shape = markertype
+        scatter = _marker_types[shape](
+            x=x, y=y, size=size, fill_color=color, fill_alpha=fill_alpha,
+            line_color=line_color, line_alpha=line_alpha
+        )
+
+        self._append_glyph(source, scatter)
+
+        return scatter
```

Comment:
A little confused by all these `make_X` functions, what is their use-case for users?